src/main.ts

```diff
@@ -148,6 +153,10 @@ export class Minifier {
     var newName = chars.join('');
     if (this.checkReserved(newName)) {
       return this.generateNextPropertyName(newName);
+      // Property names cannot start with a number. Generate next possible property name that starts
+      // with the first alpha character.
+    } else if (this.isNumeric(chars[0])) {
```

Comment:
I think it's cleaner here to test for [0-9], you don't really want to parse that as a number.